### ISSUE TITLE
frisby cannot post xml?

### ISSUE THREAD

**Issue by mthoreso** [vlucas/frisby#332]
@vlucas Is it possible to post XML data?

I cannot figure out how to .post() xml data.

When I post JSON:
```
.post(envSetup.URL + '/' + resource + '?action=basic', {
    arr: [1, 2, 3, 4],
    foo: "bar",
    bar: "baz",
    answer: 42 },
    {json: true,
    headers: {
      'content-type': 'application/json',
      'Authorization': 'Basic 02d5ffdb-eec9-43d8-b1a6-79826fa97788',
      'Accept': 'application/json'}
  })
```

The .inspectRequest() shows a 'body' with stuff in it!:
```
{ json: true,
  uri: 'http://dev.<company url>/api/rules?action=basic',
  body: { arr: [ 1, 2, 3, 4 ], foo: 'bar', bar: 'baz', answer: 42 },
  method: 'POST',
  headers:
   { 'content-type': 'application/json',
     Authorization: 'Basic <token>',
     Accept: 'application/json' },
  inspectOnFailure: false,
  baseUri: '',
  timeout: 5000 }
```

But if I .post() XML:
```
var xml_body = envSetup.ENV_DATA.inrule.xml_post_kia1;

.post(envSetup.URL + '/' + resource + '?action=basic', xml_body, {
    json: false,
    headers: {
      'content-type': 'application/xml',
      'Authorization': 'Basic 02d5ffdb-eec9-43d8-b1a6-79826fa97788',
      'Accept': 'application/xml'}
  })
```

It puts nothing in the 'body':
```
{ json: false,
  uri: 'http://dev.<company url>/api/rules?action=basic',
  body: null,
  method: 'POST',
  headers: { 'content-type': 'application/x-www-form-urlencoded' },
  inspectOnFailure: false,
  baseUri: '',
  timeout: 5000 }
```

If frisby.js can't do XML then it won't fully meet my needs (and I love using it).  I'd love if the api only did JSON, but sadly that's not how everything works.

Thanks.

**Comment by vlucas** [comment]
Can you try this with v2 and let me know if you can do it? #316

**Comment by vlucas** [comment]
Just realized this has actually been solved before - check out #290.

**Comment by mthoreso** [comment]
Awesome, thank you for that.  I should've done a better job searching the issues.
